tests/sample.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * SLA bridge and channel creation tests.
+ *
+ */
+"use strict";
+
+/*global describe:false*/
+/*global afterEach:false*/
+
+var util = require('util');
+var Emitter = require('events').EventEmitter;
+
+var mockClient;
+var mockChannel;
+var mockBridge;
+// The channels in existance
+var channels = [];
+// The bridges in existance (should only be one)
+var bridges = [];
+// The channels in the bridges
+var bridge_channels = [];
+// Tests if a bridge has been created or not on a particular test. Resets
+var bridge_created = 0;
+// Mock value for whether the endpoint is valid or not
+var valid = 1;
+// Conditional for whether or not StasisStart has been passed yet
+var pastStasis = 0;
+
+// The mocked up version of the callback error function
+var end= function(err){console.log(err);}
+
+// Millesecond delay for mock requests
+var asyncDelay = 100;
+
+/**
+ * Returns a mock client that houses the bridges and can create both bridges
+ * and channels.
+ */
+
+var getMockClient = function() {
+  if(mockClient) {
+    return mockClient;
+  }
+  var Client = function() {
+    this.bridges = {
+      list: function(cb) {
+        console.log(bridges);
+        cb(null, bridges);
+        return bridges;
+      },
+      create: function(param, cb) {
+        bridge_created = 1;
+        console.log(param);
+        bridges.push(getMockBridge(param));
+        console.log(bridges)
+        cb(null, bridges[0]);
+      }
+    };
+    this.Channel= function() {
+      return getMockChannel();
+      }
+  };
+  mockClient = new Client();
+  return mockClient;
+};
+
+/**
+ * Returns a mock bridge that has a bridge_type and id, and can add channels
+ * to its array of channels. Emits a ChannelEnteredBridge event
+ */
+
+var getMockBridge = function(param) {
+  var Bridge = function(param) {
+    console.log(param.type);
+    this.bridge_type = param.type;
+    this.id = "1";
+    this.addChannel= function(input, cb) {
+      console.log("TREX");
```

Comment:
console.log statements are ok while you are writing tests but generally do not get included as part of the tests. Tests report whether they pass or not to the terminal but usually no more.